server/accounts.go

```diff
@@ -2846,20 +2844,26 @@ func (s *Server) updateAccountClaimsWithRefresh(a *Account, ac *jwt.AccountClaim
 	// Reset any notion of export revocations.
 	a.actsRevoked = nil
 
+	alteredScope := map[string]struct{}{}
+
 	// update account signing keys
 	a.signingKeys = nil
+	if len(ac.SigningKeys) > 0 {
+		a.signingKeys = map[string]jwt.Scope{}
```

Comment:
Should we have a make here? Also, we could initialize the size to ac.SigningKeys?

Reply:
fixed